libraries/chef_pyenv_script_helpers.rb

```diff
@@ -1,29 +1,17 @@
-require 'chef/mixin/shell_out'
-
 class Chef
```

Comment:
This can be a module and I think we'd be best of standardising on a namespace that doesn't pollute the chef one imo. The pattern we've followed in other cookbooks which I think is the way to go is:
```ruby
module PyEnv
  module Cookbook
     module ScriptHelpers
       #stuff
     end
  end
end
```